src/Util/PhpCompatUtil.php

```diff
@@ -32,7 +33,7 @@ public function canUseAttributes(): bool
     {
         $version = $this->getPhpVersion();
 
-        return version_compare($version, '8alpha', '>=');
+        return version_compare($version, '8alpha', '>=') && Kernel::VERSION_ID >= 50200;
```

Comment:
We should probably make this method more specific. This is really about *routing* attributes. But soon we will probably start using Doctrine attributes and then other things. Possibly we could rename this to `canUseRouteAttributes()` and add other methods later.